test/SILOptimizer/sil_combine_objc.sil

```diff
@@ -19,18 +19,18 @@ class ZZZ {
 
 class C {}
 
-sil @stringcore_invariant_check : $@convention(thin) (@owned _StringCore) -> @owned Optional<_CocoaString>
+sil @stringcore_invariant_check : $@convention(thin) (@owned _LegacyStringCore) -> @owned Optional<_CocoaString>
```

Comment:
These sorts of tests don't have much value so if the only thing preventing you from removing _LegacyStringCore is this test, nuke the test.

Reply:
_LegacyStringCore was removed. Aha, this test is XFAILed. I'll remove the portion of the test.